libqtile/backend/wayland/core.py

```diff
@@ -774,6 +789,18 @@ def stack_windows(self) -> None:
         else:
             self.stacked_windows = self.mapped_windows
 
+    def check_idle_inhibitor(self) -> None:
+        """
+        Checks if any window that is currently mapped has idle inhibitor
+        and if so inhibits idle
+        """
+        for win in self.mapped_windows:
+            if isinstance(win, window.Window) and win.is_idle_inhibited:
```

Comment:
The type of `self.mapped_windows` is `window.WindowType` so I think this `isinstance` test is redundant.

Reply:
You are right that is redundant, fixed.